sdk/reservations/Azure.ResourceManager.Reservations/src/Generated/ReservationOrderResponseData.cs

```diff
@@ -57,7 +56,7 @@ internal ReservationOrderResponseData(ResourceIdentifier id, string name, Resour
         }
 
         /// <summary> Gets the etag. </summary>
-        public ETag? Etag { get; }
+        public int? Etag { get; }
```

Comment:
Revert this change so it doesn't block testing. The backend service returns an integer.